regression/jbmc-strings/Makefile

```diff
@@ -1,28 +1,16 @@
 default: tests.log
 
 test:
-	@if ! ../test.pl -c ../../../src/jbmc/jbmc ; then \
-		../failed-tests-printer.pl ; \
-		exit 1 ; \
-	fi
+	../test.pl -p -c ../../../src/jbmc/jbmc
```

Comment:
Please hide the output of this line by prefixing it with an `@`. (I wouldn't strictly mind, but this is how it's done in all other regression/*/Makefile.)